app/test_split_integer.py

```diff
@@ -1,21 +1,44 @@
+
 from app.split_integer import split_integer
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 15
+    number_of_parts = 3
+    parts = split_integer(value, number_of_parts)
+    assert sum(parts) == value
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    value = 12
+    number_of_parts = 3
+    parts = split_integer(value, number_of_parts)
+    assert len(parts) == number_of_parts
```

Comment:
it seems to me that the check for equality check that u do in each func is redundant